examples/run_all_examples.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
is there alternative to bash? does this work with zsh? perhaps python based script?

Reply:
run on my Zsh shell without a problem. I think that is more of a fallback.